### DISCUSSION TITLE
API documentatiom

### DISCUSSION THREAD

**Discussion by albin02t** [doccano/doccano#1944] [Q&A]
I wanted to include docanno in my python script using the REST APIs. Where can i find the api documentation? 
Accessing http://127.0.0.1:8000/swagger/ gives 403 forbidden

**Comment by albin02t** [comment]
Okay i got the documentation but trying it out in postman gives me an error for X-CSRF token. Where can i get the token?

**Comment by Vstay97** [comment]
hi, i hava  a same question (403 forbidden) 

can u tell me how to solve this question ? please @albin02t

**Reply by albin02t** [reply]
try http://localhost:8000/swagger/
That should work

**Reply by Vstay97** [reply]
Thank you very much for your help, I tried your link and it still shows "403 forbidden".

But I can use swagger normally by manually adding the chrome cookie.

Thanks again for your help!

**Comment by david-engelmann** [comment]
If you login into doccano as admin by loading up the [http://127.0.0.1:8000](http://127.0.0.1:8000) page, you'll then be able to load the */swagger url and see the endpoints available.

Just to note, my url is without the :8000, so some users will want to try that as well if they receive the `This site can’t be reached` error
